src/Microsoft.IdentityModel.Tokens/EventBasedLRUCache.cs

```diff
@@ -100,7 +101,7 @@ internal class EventBasedLRUCache<TKey, TValue> : IDisposable
 
         internal EventBasedLRUCache(
             int capacity,
-            TaskCreationOptions options = TaskCreationOptions.LongRunning,
+            TaskCreationOptions options = TaskCreationOptions.None, // default to None instead of LongRunning as LongRunning will always start a task on a new thread instead of ThreadPool
```

Comment:
I remember messing around with the value of this property caused multiple hanging unit tests for us. We'll have to make sure that isn't happening again.